modules/lightmapper_rd/config.py

```diff
@@ -1,6 +1,17 @@
 def can_build(env, platform):
-    return env.editor_build and platform not in ["android", "ios"]
+    return (env.editor_build and platform not in ["android", "ios"]) or env["module_lightmapper_rd_enabled"]
 
 
 def configure(env):
-    pass
+    from SCons.Script import BoolVariable, Variables, Help
+
+    envvars = Variables()
+    envvars.Add(
+        BoolVariable(
+            "lightmapper_rd",
+            "Enable Lightmapper functionality in export template builds (increases binary size)",
+            False,
+        )
+    )
```

Comment:
I'm confused why we need a separate `lightmapper_rd` option for this, considering we already have `module_lightmapper_rd_enabled` option that is `True` by default in editor builds and `False` by default in export template sbuilds.